packages/fxa-auth-server/lib/routes/validators.js

```diff
@@ -466,7 +466,7 @@ module.exports.subscriptionProductMetadataBaseValidator = isA
     successActionButtonURL: isA.string().uri().required(),
     appStoreLink: isA.string().uri().optional(),
     playStoreLink: isA.string().uri().optional(),
-    productSet: isA.string().optional(),
+    productSet: isA.array().items(Joi.string()).optional(),
```

Comment:
Should this just use `isA.string()` (instead of `Joi.string()`) to avoid needing to import `Joi`?

```js
isA.array().items(isA.string()).optional(),
```

Ref: https://github.com/mozilla/fxa/blob/87e34c2fa008dd4ed3407062919523e5ca038a79/packages/fxa-auth-server/lib/routes/validators.js#L736-L748